src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -924,7 +923,10 @@ protected void enqueueAndHandleVariant(final VariantContext variant, final Refer
 
         final FuncotationMap funcotationMap = funcotatorEngine.createFuncotationMapForVariant(variant, referenceContext, featureContext);
 
+        // This is necessary because we want to revert the variant contig name change if it was applied in the FuncotatorEngine::getCorrectVariantContextForReference method before outputting the vcf.
```

Comment:
This comment is still misleading -- can you make it clear that we don't *always* want to revert the variant contig name change, only in the very specific case where the input is b37 and the reference is hg19?